src/pages/Scan.vue

```diff
@@ -128,17 +121,15 @@ export default {
                     function(error) {
                         if (!Store.netMgr.online) {
                             this.showQueued();
-                            this.setMessage("Not enough signal, voucher queued.", constants.MESSAGE_ERROR);
+                            this.setMessage("Not enough signal, voucher queued.", constants.MESSAGE_WARNING);
                         }
                     }.bind(this));
 
                 // Do anyway.
                 this.voucherCode = "";
-                this.sponsorCode = "";
```

Comment:
@ob6160 yeah it was probably a mistake. As I said, add them back in if you need to